trpcultivate_phenotypes/tests/src/Kernel/Services/Traits/ValidTraitTest.php

```diff
@@ -55,7 +55,7 @@ class ValidTraitTest extends ChadoTestKernelBase {
    * Term config key to cvterm_id mapping.
    * Note: we just grabbed some random cvterm_ids that we know for sure exist.
    */
-  protected array $terms = [
+  protected $terms = [
```

Comment:
Just removes this change as I seem to remember it being editor focused and not needed.
```suggestion
  protected array $terms = [
```